things/api/things/http/endpoint_test.go

```diff
@@ -35,6 +35,7 @@ const (
 	nameKey     = "name"
 	ascKey      = "asc"
 	descKey     = "desc"
+	Prefix      = "fe6b4e92-cc98-425e-b0aa-"
```

Comment:
Why this const is exported? Can we maybe use `uuidPrefix`?

Reply:
Ideally it should work when we use uuid.Prefix. This internal uuid.Prefix is used when we do not supply any external UUID. 
I was preferring to give an emphasis to show that we are using an external UUID.
Let me know your thought.  I am currently leaving it as it is and resolving it.

Reply:
There is no reason to export this constant if you are only using it in this package. Call it `prefix` or `uuidPrefix` please.

Reply:
Now I realised your point :) There is no need to export Prefix.  Corrected and pushed, here and in things/service_test.go